### PR TITLE
Xorriso

### PR DESCRIPTION
**Pull request by sergio-chainguard**
xorriso

There are package dependencies to build this package, they are provided in the files.

https://github.com/chainguard-dev/image-requests/issues/869

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: libburn.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      runtime:
+        - libburn
+    description: libburn dev
+
+  - name: libburn-doc
+    pipeline:
+      - uses: split/manpages
+    description: libburn manpages
+
+update:
```
Comment (mamccorm) [inline libburn.yaml:48]: Good news - looks like this is registered in release monitor:
- https://release-monitoring.org/project/1568/

So we could enable auto-updates here. Theres some other examples in wolfi we can refer to:
- https://github.com/search?q=repo%3Awolfi-dev%2Fos%20release-monitor%3A&type=code

File: libisoburn.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    description: libisoburn manpages
+
+  - name: xorriso
+    pipeline:
+      - runs: |
+          make install DESTDIR=${{targets.subpkgdir}}
+          mkdir -p ${{targets.subpkgdir}}
+          ln -sf /usr/bin/xorriso ${{targets.subpkgdir}}/xorriso
+
+update:
```
Comment (mamccorm) [inline libisoburn.yaml:59]: Similar for here :) 
- https://release-monitoring.org/project/1645/

File: libisofs.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+subpackages:
+  - name: libisofs-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libisofs
+    description: libisofs dev
+
+update:
```
Comment (mamccorm) [inline libisofs.yaml:44]: Similar for here :) 
https://release-monitoring.org/project/1646/

**Review by mamccorm — APPROVED** [review]
looks good

File: libburn.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libburn/APKBUILD
+package:
+  name: libburn
```
Comment (Dentrax) [inline libburn.yaml:3]: we can instead use single manifest with package name `libburnia` then we can handle with `data.items` pipeline:

```
data:
  - name: components
    items:
      libburn: libburn description
      libisoburn: libisoburn description
      libisofs: libisofs description
```

example here: https://github.com/wolfi-dev/os/blob/b1dc8fb0c9370fb450c1ef4a202fbea7380ba619/cfssl.yaml#L36-L38

**Pull request merged**